addons/mail/static/src/rtc/call.js

```diff
@@ -38,7 +38,7 @@ export class Call extends Component {
             columnCount: 0,
             overlay: false,
         });
-        this.store = useStore();
+        this.discussStore = useService("discuss.store");
```

Comment:
Same comment: accessing `discuss` from outside its folder is not correct.

Reply:
same comment: I'll move the whole rtc folder to discuss.